src/app/CMakeLists.txt

```diff
@@ -566,6 +566,7 @@ ENDIF(MSVC)
 
 INCLUDE_DIRECTORIES(
   ${CMAKE_SOURCE_DIR}/external/libdxfrw
+  ${CMAKE_SOURCE_DIR}/external/nmea
```

Comment:
i'm wondering - should these external libs be included as SYSTEM directories? I understand the difference is mainly about which warnings to throw when building, and we don't really care about warnings for these libs.

Reply:
> these fixed-install system files not being considered in dependency calculations

do we really want this?